src/CircleCI-MinGW.sh

```diff
@@ -180,7 +180,7 @@ echo "now building/testing a 32 bit SSE2 Linux build"
 echo '******************************************************************************'
 echo ""
 make -f Makefile.legacy -s clean
-JOHN_CFLAGS=-m32 JOHN_ASFLAGS=-m32 JOHN_LDFLAGS=-m32 make -f Makefile.legacy -sj4 linux-x86-sse2
+JOHN_CFLAGS=-m32 JOHN_ASFLAGS=-m32 JOHN_LDFLAGS=-m32 make -f Makefile.legacy -j4 linux-x86-sse2
 # do NOT exit on error from make.  We expect an error in the libpcap stuff
```

Comment:
Did you really want the changes above to be permanent? If so, I don't mind - just asking.